src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Bulldozer extends  Machine{
+
+    @Override
+    public void doWork () {
+
+        System.out.println("Bulldozer started to work!");
```

Comment:
```suggestion
    public void doWork () {
        System.out.println("Bulldozer started to work!");
```
між сигнатурою методу і тілом зазвичай не ставиться пуста лінія